### PR TITLE
Add support to  expand animation slide while downloading courses

### PR DESCRIPTION
**Pull request by devKshitijJain**
Fixes- https://github.com/shihabmridha/educative.io-downloader/issues/10

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -157,6 +157,13 @@ async function downloadPage(title: string, link: string, doToggleMenu: boolean):
 
   await page.addStyleTag({ content: 'div[class^="styles__PrevNextButtonWidgetStyled"], div[class^="styles__Footer"], nav { display: none !important; }' });
 
+  await page.evaluate(() => {
+    const elements = document.getElementsByClassName('Button-ktk5iw-0 CircleButton-f4i43u-0 styles__AnimationPlus-sc-2az9w3-13 dqWxjg')
```
Comment (shihabmridha) [inline src/app.ts]: I believe this class name is dynamic. It might not be same for everyone. So, it would be safe to handle using node.

Comment (devKshitijJain) [inline src/app.ts]: Is there any better way to define this class name? I mean as you have done above class^="styles__PrevNextButtonWidgetStyled"

Comment (devKshitijJain) [reply src/app.ts]: @shihabmridha just commented same. Can you please look into it. I am not so sure about it.

Comment (shihabmridha) [reply src/app.ts]: That one is hardcoded in their app. But, the one you are using seems like a dynamically generated by react.

File: src/app.ts [hunk 2]
```diff
@@ -157,6 +157,17 @@ async function downloadPage(title: string, link: string, doToggleMenu: boolean):
 
   await page.addStyleTag({ content: 'div[class^="styles__PrevNextButtonWidgetStyled"], div[class^="styles__Footer"], nav { display: none !important; }' });
 
+  await page.evaluate(() => {
+    var xPathResult = document.evaluate('//button[contains(@class, "AnimationPlus")]', document, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
```
Comment (devKshitijJain) [inline src/app.ts]: Changed to Dynamic depending on text AnimationPlus

Comment (shihabmridha) [reply src/app.ts]: Worked nicely!
One suggestion, Please change the `var` to `const`. Same for line 163 (`let` to `const`).

**Review by shihabmridha — CHANGES_REQUESTED** [review]

**Review by shihabmridha — APPROVED** [review]

**Pull request merged**